packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -63,12 +63,7 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
     }
 
     protected submitCore(type: string, messages: IDocumentMessage[]) {
-        if (this.isBatchManagerDisabled) {
-            this.emitMessages(type, [messages]);
-        } else {
-            this.submitManager.add(type, messages);
-            this.submitManager.drain();
-        }
+        this.emitMessages(type, [messages]);
```

Comment:
Can you please open a follow-up bug to rething protocol? There is no need to have array of array ops any more, I believe. That said, possibly we want to delay any such work until we get rid of notion of batches in protocol altogether, and just say - it's an array of loosely connected messages :)

Reply:
https://github.com/microsoft/FluidFramework/issues/9322